rp2040-hal/src/prelude.rs

```diff
@@ -1,3 +1 @@
 //! Prelude
-pub use crate::gpio::GpioExt;
-pub use crate::sio::Sio;
```

Comment:
Removed this as I realized this should never have been here. The prelude should really be traits only.